js/eclipsefdn.adopters.js

```diff
@@ -11,25 +11,10 @@
  * SPDX-License-Identifier: EPL-2.0
 */
 
-(function(root, factory) {
-  if (typeof define === 'function' && define.amd) {
-    define(['efa'], factory(root));
-  }
-  else if (typeof exports === 'object') {
-    module.exports = factory(require('efa'));
-  }
-  else {
-    root.eclipseFdnAdopters = factory(root, root.efa);
-  }
-})(this, function(efa) {
-
-  'use strict';
-
-  // Define object
-  var eclipseFdnAdopters = {};
-  var precompiledRegex = /<([^>]*?)>;(\s?[\w-]*?="(?:\\"|[^"])*";){0,}\s?rel="next"/;
+class EclipseFdnAdopters {
```

Comment:
@autumnfound I just noticed that you don't need a constructor here 🤔 Seems it will auto-generate an empty one which has no effects on this?

Reply:
It assumes an empty constructor yes. That's the default in most languages.

Reply:
IC. I never created a class without constructor 😂 thx!